crates/bevy_input/src/gamepad.rs

```diff
@@ -108,12 +107,27 @@ impl Gamepad {
     reflect(Serialize, Deserialize)
 )]
 pub struct GamepadInfo {
-    /// The name of the gamepad.
+    /// The display name of the gamepad.
     ///
-    /// This name is generally defined by the OS.
-    ///
-    /// For example on Windows the name may be "HID-compliant game controller".
+    /// This name is generally defined by the OS. For example on Windows the
+    /// name may be "HID-compliant game controller". This is a display name,
+    /// with no guarantee of unicity; if a user has two devices from the same
+    /// make and model, chances are this name will be the same. For a unique
+    /// identifier stable across runs, see [`uuid`] instead.
```

Comment:
```suggestion
    /// This name is defined by the OS. For example on Windows the
    /// name may be "HID-compliant game controller". The name is not unique,
    /// typically two gamepads of the same model will have the same name.
    /// For a unique identifier stable across runs, see [`uuid`] instead.
```

Reply:
We use `Gamepad::name()` not `Gamepad::os_name()`, so `gilrs` might not return the OS name always, hence the "generally". Should we switch to `os_name()`?

https://docs.rs/gilrs/latest/gilrs/struct.Gamepad.html

Reply:
Left the "generally", applied the other changes.